indexer/bin/indexer.js

```diff
@@ -1,3 +1,4 @@
+import { handleProviderRegistered } from '../lib/handle-provider-events.js'
```

Comment:
We use nouns, not verbs, for naming files.

```suggestion
import { handleProviderRegistered } from '../lib/provider-events-handler.js'
```

Reply:
This is new to me (sorry if I didn't notice), I had proposed that file name.

Reply:
Resolved in 19acee8